llvm/lib/ObjCopy/ELF/ELFObjcopy.cpp

```diff
@@ -290,6 +290,9 @@ static Error updateAndRemoveSymbols(const CommonConfig &Config,
     return Error::success();
 
   Obj.SymbolTable->updateSymbols([&](Symbol &Sym) {
+    if (ELFConfig.SymbolsToSetVisibility.matches(Sym.Name) &&
```

Comment:
At first I thought that it would make sense to first set some symbols to hidden and then localize them with --localize-hidden, but I guess if someone already knows the symbol names they can just use --localize-symbols and then set their visibility to hidden if needs be.